src/test/java/fr/lunatech/timekeeper/resources/EventResourceTest.java

```diff
@@ -71,46 +72,57 @@ void cleanUp() {
 
```

Comment:
I expect to see a test which verify that you get a 400 when you try to create or update an event with a same name, startDate and endDate.
To do that, you can use postValidation and putValidation